assets/extensions/main.js

```diff
@@ -60,6 +63,12 @@ const Main = () => {
 	const installedExtensions = extensions.filter(
 		( extension ) => extension.is_installed
 	);
+	const wooExtensions = extensions.filter(
```

Comment:
For `wooExtensions`, and `nonWooExtensions`, we could add a new argument to filter it in the `getExtensions` selector (also in the other branch only).